### DISCUSSION TITLE
Pelorus now has support for ConfigMap based configuration

### DISCUSSION THREAD

**Discussion by weshayutin** [dora-metrics/pelorus#492] [Show and tell]
Tell us what you think!
https://github.com/konveyor/pelorus/blob/master/docs/Configuration.md#configmap-configuration-values

**Comment by mateusoliveira43** [comment]
closing as this is documented https://pelorus.readthedocs.io/en/latest/GettingStarted/configuration/PelorusExporters/#configmaps
